env.sh

```diff
@@ -1,11 +1,17 @@
 #!/usr/bin/env bash
 
 if [ `uname` == 'Darwin' ]; then
+    SYSTEM_NET="bridge"
```

Comment:
We don't need expose ports if we just use this script to build CITA.

Reply:
#400

Reply:
I mean we do not need to expose ports in `env.sh`, because this script just use to build `CITA`.
So we do not need this line, and 

```
 --net=${SYSTEM_NET} \

-p $EXPOSE ${DOCKER_IMAGE} \
```

Reply:
should I remove the `SYSTEM_NET` condition, and use `bridge` by default following #400 ?

Reply:
no.

we now have only one script.

https://github.com/cryptape/cita/blob/develop/scripts/release.sh#L58

Reply:
> Should I remove the `SYSTEM_NET` condition, and use `bridge` by default following #400 ?

Please do not do it in this pr.

Reply:
Yeah, I have got this. Just forgot what i said.